torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -534,6 +534,16 @@ static void PrepareListAppendAndInsertForONNX(Block* b) {
   }
 }
 
+static void ChangeInputTypeForInplaceOps(Node* inplaceNode) {
+  // TODO: Check if binary op
+  auto newInputNode = inplaceNode->owningGraph()->create(aten::type_as, 1);
+  auto orignalInputs = inplaceNode->inputs();
+  newInputNode->insertBefore(inplaceNode);
+  newInputNode->addInput(orignalInputs[1]);
```

Comment:
replace `[idx]` with `at(idx)` to avoid segfaults

Reply:
good point, will make this change